### DISCUSSION TITLE
Divide fields into pages

### DISCUSSION THREAD

**Discussion by ezzadeen** [rjsf-team/react-jsonschema-form#3318] [Ideas]
When there are many fields in a json schema, it would be nice to set a page number in the UI schema for each field or just the ancestor when there is a hierarchy. The form then can look like a wizard with page navigation. The form can show each field in the corresponding page and hide the rest.

**Comment by leetwito** [comment]
rjsf is made for forms, and doesn't support wizards (multi-step)
Though I agree it is a natural extension

**Comment by ezzadeen** [comment]
I did this using the classNames in the UI schema. I can show/hide fields based on the page selection. I ended up with several UI schemas one per page.
